frontend/src/text-animation.css

```diff
@@ -0,0 +1,143 @@
+@import url('https://fonts.googleapis.com/css2?family=IM+Fell+English&display=swap');
+
+.typing {
+    font-size: 48px;
+    text-align: center;
+    font-family: 'IM Fell English', serif;
+    color: #fff;
+    white-space: nowrap;
+    overflow: hidden;
+    animation: typing 5s steps(14, end), blink-caret 0.75s step-end infinite;
+    display: inline-block;
+    text-align: justify;
+    width: auto;
+}
+
+.typing {
+  font-size: 3em;
+  animation: typing 2s steps(22), blink .5s step-end infinite alternate;
+}
+
+@keyframes typing {
+  from { width: 0; }
+  to { width: 100%; }
+}
+
+@keyframes blink {
+  from { border-right-color: transparent; }
+  to { border-right-color: rgb(185, 170, 135); }
+}
+
+@keyframes typing {
+    from { width: 0; }
+    to { width: 100%; }
+}
+
+
+  @keyframes text {
+    0%  { content: 'L'; }
+    2%  { content: 'Lo'; }
+    4%  { content: 'Lor'; }
+    6%  { content: 'Lore'; }
+    8%  { content: 'Lorem'; }
+    10%  { content: 'Lorem '; }
+    12%  { content: 'Lorem i'; }
+    14%  { content: 'Lorem ip'; }
+    16%  { content: 'Lorem ips'; }
+    18%  { content: 'Lorem ipsu'; }
+    20%  { content: 'Lorem ipsum'; }
+    22%  { content: 'Lorem ipsum '; }
+    24%  { content: 'Lorem ipsum d'; }
+    26%  { content: 'Lorem ipsum do'; }
+    28%  { content: 'Lorem ipsum dol'; }
+    30%  { content: 'Lorem ipsum dolo'; }
+    32%  { content: 'Lorem ipsum dolor'; }
+    34%  { content: 'Lorem ipsum dolor '; }
+    36%  { content: 'Lorem ipsum dolor s'; }
+    38%  { content: 'Lorem ipsum dolor si'; }
+    40%  { content: 'Lorem ipsum dolor sit'; }
+    42%  { content: 'Lorem ipsum dolor sit '; }
+    44%  { content: 'Lorem ipsum dolor sit a'; }
+    46%  { content: 'Lorem ipsum dolor sit am'; }
+    48%  { content: 'Lorem ipsum dolor sit ame'; }
+    50%  { content: 'Lorem ipsum dolor sit amet'; }
+    52%  { content: 'Lorem ipsum dolor sit amet '; }
```

Comment:
This might be easier to implement in javascript? If not, then this is fine.

Reply:
Ima keep it a stack idk how to change it into javascript if I do figure it out I will do it though